README.md

```diff
@@ -133,9 +105,20 @@ The call to `appmetrics.monitor()` starts the data collection agent, making the
 
 You should start your application using the `node` command as usual (**not** `node-hc`).
 
-You must call `require('appmetrics');` *before* the require statements for any npm modules you want to monitor. Appmetrics must be initialized first so that it can instrument modules for monitoring as they are loaded. If this is a problem due to the structure of your application you can require the module on the node command line with -r to make sure it is pre-loaded:
+You must call `require('appmetrics');` *before* the require statements for any npm modules you want to monitor. Appmetrics must be initialized first so that it can instrument modules for monitoring as they are loaded. If this is a problem due to the structure of your application you can require the module on the node command line with -r or --require to make sure it is pre-loaded:
 
-`> node -r appmetrics myapp.js`
+```sh
+$ node --require appmetrics myapp.js
+```
+or
+```sh
+$ node --require appmetrics/start myapp.js
+```
+or
+```sh
+$ export NODE_OPTIONS="require appmetrics/start"
```

Comment:
Same as above, also does this all need to be repeated?

Reply:
I've consolidated the two sections now